serialized-lob/src/main/java/com/iluwatar/serializedlob/PostGresSerializer.java

```
@@ -0,0 +1,91 @@
+ * THE SOFTWARE.
+ */
+package com.iluwatar.serializedlob;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.iluwatar.serializedlob.PostGresDatabase.connection;
+
+/**
+ *  Creates a single Character Large object(CLOB) from PostgreSQL graph of objects
+ */
+public class PostGresSerializer {
+    public static String graphToLOB(Connection c, String tableName){
+        StringBuilder lob = new StringBuilder();
+        Statement stmt = null;
+        String sql = "\\d+ " + tableName;
+        Map<String,String> columns = new HashMap<>();
+        try{
+            ResultSet columnSets = stmt.executeQuery(sql);
+            String col = "";
+            String type = "";
+
+            while(columnSets.next()){
+                col = columnSets.getString("columns");
+                type = columnSets.getString("Type");
+                columns.put(col,type);
+            }
+            columnSets.close();
+
+            int size = columns.size();
+            lob.append(size);
+            for(String s:columns.keySet()){
+                lob.append(" ").append(s);
+                lob.append(" ").append(columns.get(col));
+            }
+
+        } catch (Exception e){
+            System.err.println( e.getClass().getName()+": "+ e.getMessage() );
+            System.exit(0);
+        }
+
+        sql = "SELECT * FROM " + tableName;
+        try{
+            ResultSet graph = stmt.executeQuery(sql);
+
+            while(graph.next()){
+                for (String s : columns.keySet()){
+                     lob.append(" ").append(graph.getString(s));
+                }
+            }
+
+            graph.close();
+        }catch (Exception e){
+            System.err.println( e.getClass().getName()+": "+ e.getMessage() );
+            System.exit(0);
+        }
+
+        return lob.toString();
+    }
+
+    public static void main(String[] args) {
+    }
```

- Use try-with-resources to autoclose resources
- Explain with comments what happens in the code
- System.exit is probably not the correct solution. Maybe just let the method throw the exception?